lib/commands/ping.js

```diff
@@ -0,0 +1,43 @@
+'use strict'
+
+const npmConfig = require('../config.js')
+const libnpm = require('libnpm')
+const figgyPudding = require('figgy-pudding')
+const log = require('npmlog')
+
+module.exports = ping
+
+function ping (argv) {
+  const {json, registry, silent} = argv
+
+  const PingConfig = figgyPudding({
+    json: {
+      default: json
+    },
+    registry: {
+      default: registry
```

Comment:
You can remove this entire option. `npm-registry-fetch` will pick it up off `argv` (see below)

Reply:
By remove it do you mean remove the entire `registry` property or just remove the default?

Reply:
Sorry, I misspoke: I didn't remember you needed to actually access it below. Just remove the default. :)